### DISCUSSION TITLE
Any way to disable "Show Preview" globally for all collections always...

### DISCUSSION THREAD

**Discussion by rchrdnsh** [sveltia/sveltia-cms#161] [Q&A] [answered]
Cannot find anything in the docs to globally disable "Show Preview" since nobody wants it for the client I am working with.

Thank you!

**Comment by kyoshino** [comment, marked as answer]
The Decap doc is not clear, but the [`editor`](https://decapcms.org/docs/configuration-options/#editor) collection option can also be used as a global option:

> Note: Setting this as a top level configuration will set the default for all collections

```yaml
editor:
  preview: false
```
